This patch disables display driver in u-boot which improves EDID detection
https://forum.armbian.com/topic/13651-pine-a64-no-hdmi-signal/?do=findComment&comment=99169

diff --git a/configs/sunvell_r69_defconfig b/configs/sunvell_r69_defconfig
index d03dcec8..f1fc8f12 100644
--- a/configs/sunvell_r69_defconfig
+++ b/configs/sunvell_r69_defconfig
@@ -17,4 +17,4 @@ CONFIG_USB_OHCI_HCD=y
 CONFIG_SYS_USB_EVENT_POLL_VIA_INT_QUEUE=y
 CONFIG_MMC_SUNXI_SLOT_EXTRA=2
 CONFIG_SYS_CLK_FREQ=480000000
-
+CONFIG_VIDEO_DE2=n
